sbb/agora2mets.pl

```diff
@@ -0,0 +1,488 @@
+#!/opt/ActivePerl-5.16/bin/perl
```

Comment:
ActivePerl is typical for Windows environment, but typical Linux distributions use /usr/bin/perl.